web/ui/mantine-ui/src/pages/query/DataTable.tsx

```diff
@@ -161,4 +204,44 @@ const DataTable: FC<DataTableProps> = ({ expr, evalTime, retriggerIdx }) => {
   );
 };
 
+export const histogramTable = (h: Histogram): ReactNode => (
+  <Table>
+    <Table.Thead>
+      <Table.Tr>
+        <Table.Th style={{ textAlign: "center" }} colSpan={2}>
+          Histogram Sample
```

Comment:
The best title for this table is maybe "Bucket counts" now, since you have the total sum + count above the table. But probably it's even ok / better to just omit the title, hopefully the table should be clear.